vivarium/library/fasta.py

```diff
@@ -1,5 +1,6 @@
 from vivarium.library.path import open_relative
 
+
```

Comment:
I think this library should be moved to `vivarium-cell`. vivarium-core does not need fasta... Issue added in #44, I'll take care of this soon.